eng/scripts/Language-Settings.ps1

```diff
@@ -243,8 +243,8 @@ function ValidatePackagesForDocs($packages) {
   $validationOutput = $packages | Foreach-Object -Parallel {
     # Get value for variables outside of the Foreach-Object scope
     $scriptRoot = "$using:scriptRoot"
-    $workingDirectory = "$using:tempDirectory"
-    return ."$scriptRoot\validate-docs-package.ps1" -Package $_ -WorkingDirectory $workingDirectory
+    $imageId = "$using:ImageId"
```

Comment:
Where is this ImageId coming from?

Reply:
What happens if `$ImageId` is empty?

Reply:
Will add a check for this in common script.